assets/scss/components/_card.scss

```diff
@@ -0,0 +1,83 @@
+/*doc
+---
+title: Cards
+name: uiCard
+category: UI Components
+---
+Card-like boxes.
+
+## Standard, Group, and Event cards
+(This example uses `gridList`, but cards may placed anywhere)
+
+```html_example
+<section class="stripe--collection">
+	<div class="bounds">
+		<ul class="gridList gridList--has1 atMedium_gridList--has3">
+			<!-- Base card styles -->
+			<li class="gridList-item">
+				<div class="card">
+					<h4>Look at this card</h4>
+					<p>Just <em>look at it</em></p>
+				</div>
+			</li>
+			<li class="gridList-item">
+				<div class="card">
+					<h4>Look at this card</h4>
+					<p>Just <em>look at it</em></p>
+				</div>
+			</li>
+			<li class="gridList-item">
+				<div class="card">
+					<h4>Look at this card</h4>
+					<p>Just <em>look at it</em></p>
+				</div>
+			</li>
+		</ul>
+	</div>
+</section>
+```
+*/
+
+.card {
+	border: .5px solid $C_border; // change to $C_borderCard when constants branch merges
+	box-sizing: border-box;
+	display: block;
+	background: $C_contentBG;
+	border-radius: $largeRadius;
+	position: relative;
+	background-size: cover;
+	background-clip: padding-box;
+	min-height: $block-4;
+
+	@include responsiveVarContext--base() {
+		padding: $base $base 0;
+	}
+}
+
+.card--initialHeight {
+	min-height: initial;
+}
+
+.card--hasShadow {
+	@extend %shadow;
+
+	&.card--hasHoverShadow{
+
+		&:hover,
+		&:focus {
+			box-shadow: 0 0 4px 0 $C_border, 0 4px 12px 0 $C_border;
+		}
+
+	}
+
+}
+
+.card--hasHoverShadow {
```

Comment:
Should these hover states have a default `transition` property? The effect on hover is a little sudden.